src/index.html

```diff
@@ -1,12 +1,44 @@
 <!DOCTYPE html>
-<html lang="en">
+<html lang="ru">
   <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Calendar</title>
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
-  <body>
-    <h1>Calendar</h1>
+  <body class="container">
+    <main class="month month--month-length_28">
+      <div class="month__day month__day--1 month__day--start-day_sat"></div>
+      <div class="month__day month__day--2"></div>
```

Comment:
```suggestion
      <div class="month__day"></div>
```
Тут хватит только этого класса, все остальное можно делать через nth-child